zkevm-circuits/src/copy_circuit.rs

```diff
@@ -457,12 +487,37 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                 meta.query_advice(value, CURRENT),
             ]
             .into_iter()
-            .zip_eq(bytecode_table.table_exprs_mini(meta))
+            .zip_eq(table_expr)
             .map(|(arg, table)| (cond.clone() * arg, table))
             .collect()
         });
 
-        meta.lookup_any("rw lookup", |meta| {
+        // lookup second bytecode table
+        #[cfg(feature = "dual_bytecode")]
+        meta.lookup_any("Bytecode lookup", |meta| {
```

Comment:
May be its better to use a different name here to diffrentiate between which lookup failed:
```suggestion
        meta.lookup_any("Bytecode1 lookup", |meta| {
```